ember.js/2017-01/january-06.md

```diff
@@ -0,0 +1,163 @@
+
+YK: I've been trying to make my own work more incremental.
+
+SP: Yeah the recent stuff with opcodes was good, but the surprise was the learna stuff causing extra work.
+
+YK: the stack-based work I'm doing now is less incremental. It isn't always obvious how you can make work incremental.
+
+####  Frozen hash and params bugfix into stable
+
+MB: This has landed now but we have not cut a release recently. Are we going to just put this in 2.11 and skip backporting to 2.10?
+
+RJ: I think it will just roll into 2.11. I haven't heard anyone clamoring for a 2.10 backport. This kind of thing is what LTS releases for, and the issue is not in 2.8.
+
+KS: If people can't incrementally move from 2.9 to 2.10 to 2.11, then that's a stronger argument for fixing it in 2.10.
+
+RJ: There's not a lot of big differences between 2.10 and 2.11 that would cause people trouble moving between them.
+
+KS: We need a way to deprecate/strip things in our microlibraries as well. We should extract utilities for doing deprecations and production builds/stripping.
+
+RJ: There are some problems with the built-in debug utilities that we should work on that could probably done with this work.
+
+KS: Before we talk about this more, Matthew, are there any action items with your agenda item we need to discuss?
+
+MB: I will follow up on the GitHub issue where people are waiting for a fix, if no one is strongly opposed we will just ask people to upgrade to 2.11.
+
+#### nursery and retirement home
+
+RJ: RM raised this idea
+
+RM: I have two things, the nursery and the retirement home. Rust does both and Dockyard has davy jone's locker. Ember-rails is an example, only one person works on it.
+
+YK: I think making Ember work with Rails is actually important. And we should not just remove it.
+
+RM: There is also ember-cli-rails
+
+RJ: Right I would start there if I needed to today.
+
+RM: This is part of my learning queue. To write integration cookbooks. But I need help since I don't want to only have Rails.
+
+YK: We should not that Rails remains very popular. If we remove it people will think the opposite of what we want, which is that Rails is very important.
+
+SP: I think when we look to deprecate we should need to understand the alternative and document it (ala Ember deprecations)
+
+TD: Isn't the Rails integration just JSON-API?
+
+YK: No there are a few other things needed to get up and running. We've had interns who didn't know where to start. Copy/paste these ten commands would be enough. Or a Gem + API.
+
+TD: Let's move ahead with the retirement home and worry about the nursery when we have something to put in it.
+
+(some discussion about Rails + Ember strategy)
+
+#### ember-cli update
+
+SP: The ES class stuff in Node 4 is compat with our core object, so that is great. Bower is gone on master (the system version would be used if an addon needs it), npm is nearly optional. At some point in the future we can make the swap to yarn as well.
```

Comment:
@stefanpenner that's actually not quite correct. we don't use global bower, but instead install bower lazily if needed. using global bower is hard because we `require()` it for interactive mode instead of using `execa`.